src/GT.Data/Data/GTAppDb/Entities/Listing.cs

```diff
@@ -33,5 +33,8 @@ public class Listing : IGTEntity
 		public ICollection<ListingInquiry>? Inquiries { get; set; }
 
 		public ExperienceLevel? ExperienceLevel { get; set; }
+
+		public DateTime? ApplicationDeadline { get; set; }
+		public string? CompanyLogoId { get; set; }
```

Comment:
Should maintain line breaks between each property for conformity with the rest of the document.

Reply:
Fixed formatting